### PR TITLE
Fix clockwork

### PR DESCRIPTION
**Pull request by jessieay**
* Was a typo in class name
* Also refine losing bidder email logic
* Was not checking that a losing bidder was not ALSO a winning bidder

### CODE REVIEW

**Comment by jessieay** [comment]
code climate is failing for reasons unrelated to this PR. going to update code climate config in a separate PR

File: config/clock.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@
 module Clockwork
   every(1.day, "losing_bidder_emails.send", at: "03:00", tz: "UTC") do
     puts "Sending losing bidder emails"
-    AuctionsCLosedYesterdayFinder.new.perform do |auction|
+    AuctionsClosedYesterdayFinder.new.perform do |auction|
```
Comment (harrisj) [inline config/clock.rb]: Doh!

Comment (jessieay) [inline config/clock.rb]: srsly

File: app/services/losing_bidder_email_sender.rb [hunk 2]
```diff
@@ -4,26 +4,28 @@ def initialize(auction)
...
-    losing_bids.includes(:bidder).select do |bid|
-      bid.bidder.email.present?
+  def losing_bidders_with_emails
+    losing_bidders.select do |bidder|
+      bidder.email.present?
     end
   end
 
-  def losing_bids
+  def losing_bidders
```
Comment (harrisj) [inline app/services/losing_bidder_email_sender.rb]: I like this better, good job

**Comment by harrisj** [comment]
Looks good to me. Merging.

**Pull request merged**